core/trino-main/src/test/java/io/trino/sql/planner/TestInsert.java

```diff
@@ -197,6 +220,16 @@ private Session withPreferredPartitioning()
                 .build();
     }
 
+    private Session withPreferredPartitioningThreshold()
+    {
+        return Session.builder(getQueryRunner().getDefaultSession())
+                .setSystemProperty(USE_PREFERRED_WRITE_PARTITIONING, "true")
+                .setSystemProperty(PREFERRED_WRITE_PARTITIONING_MIN_NUMBER_OF_PARTITIONS, "2")
+                .setSystemProperty(REDISTRIBUTE_WRITES, "false")
```

Comment:
We should use default value for `REDISTRIBUTE_WRITES`, see: https://github.com/trinodb/trino/pull/7034